packages/truffle-core/lib/commands/develop.js

```diff
@@ -52,35 +53,13 @@ var command = {
 
     var config = Config.detect(options);
 
-    // testrpc account info
-    // HACK: accounts is hardcoded because it will not change for the mnemonic,
-    // and not hardcoding it would require an HTTP request
-    var mnemonic = "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat";
```

Comment:
If only there were a cryptographically secure way to automatically generate random sweet-themed mnemonics :(